src/components/SortLink.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC, useCallback } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SortBy } from '../types/SortBy';
+import { getSearchWith } from '../utils/searchHelper';
+
+type Props = {
+  sortBy: SortBy,
+  isSortingNow: boolean,
+};
+
+export const SortLink: FC<Props> = ({ sortBy, isSortingNow }) => {
+  const [searchParams] = useSearchParams();
+  const currentSort = searchParams.get('sort');
+  const currentOrder = searchParams.get('order');
+
+  const handleSortParams = useCallback(
```

Comment:
The 'type' parameter in the handleSortParams function is redundant. You can use the 'sortBy' prop directly inside the function.